src/Compilers/Core/Portable/CodeGen/StateMachineStates.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+
+namespace Microsoft.CodeAnalysis.CodeGen;
+
+internal static class StateMachineStates
+{
+    internal const int FirstIteratorFinalizeState = -3;
+
+    internal const int FinishedStateMachine = -2;
+    internal const int NotStartedStateMachine = -1;
+    internal const int FirstUnusedState = 0;
+
+    /// <summary>
+    /// First state in async state machine that is used to resume the machine after await.
+    /// </summary>
+    internal const int FirstResumableAsyncState = 0;
+
+    internal const int InitialIteratorState = 0;
+
+    /// <summary>
+    /// First state in iterator state machine that is used to resume the machine after yield return.
+    /// Initial state is not used to resume state machine that yielded.
+    /// </summary>
+    internal const int FirstResumableIteratorState = InitialIteratorState + 1;
+
+    /// <summary>
+    /// Used for async-iterators to distinguish initial state from running state (-1) so that DisposeAsync can throw in latter case.
```

Comment:
>running state [](http://example.com/codeflow?start=67&length=13)

Is this "NotStartedStateMachine" above? If so, consider changing the terms in the comment and constant to match.

Reply:
Renamed to `RunningState`, updated comment.